scripts/dokkasaurus.sh

```diff
@@ -0,0 +1,16 @@
+./gradlew dokkaHtmlMultiModule
+git clone git@github.com:GetStream/DokkasaurusSidebar.git
```

Comment:
What about to use a concrete version of this repository?
We can create some tags on this GH Repository and be sure we are using this version